test/BuildingRegistry.Tests/ProjectionTests/Wfs/BuildingUnitV2Tests.cs

```diff
@@ -1054,6 +1055,89 @@ await Sut
                 });
         }
 
+        [Fact]
+        public async Task WhenBuildingUnitWasTransferred()
+        {
+            var plannedBuildingUnit = _fixture.Create<BuildingUnitWasPlannedV2>();
+
+            var buildingPersistentLocalId = new BuildingPersistentLocalId(1);
+            var oldBuildingPersistentLocalId = new BuildingPersistentLocalId(plannedBuildingUnit.BuildingPersistentLocalId);
+            var buildingUnitPersistentLocalId = new BuildingUnitPersistentLocalId(plannedBuildingUnit.BuildingUnitPersistentLocalId);
+
+
+            var @event = new BuildingUnitWasTransferred(
```

Comment:
I've added an BuildingUnitWasTransferredBuilder in the last commit. You can use this one?